rswag-specs/lib/rswag/specs/request_factory.rb

```diff
@@ -106,7 +106,7 @@ def base_path_from_servers(swagger_doc, use_server = :default)
         server = swagger_doc[:servers].first
         variables = {}
         server.fetch(:variables, {}).each_pair { |k,v| variables[k] = v[use_server] }
-        base_path = server[:url].gsub(/\{(.*?)\}/) { |name| variables[name.to_sym] }
+        base_path = server[:url].gsub(/\{(.*?)\}/) { |name| variables[name.tr('{}','').to_sym] }
```

Comment:
Why not use the `$1` result filled with the regex's group ?
```suggestion
        base_path = server[:url].gsub(/\{(.*?)\}/) { variables[$1.to_sym] }
```
it remove the need to use the `tr`